.github/workflows/test.yml

```diff
@@ -50,4 +50,3 @@ jobs:
 
       - name: Generate coverage
         run: pnpm coverage
```

Comment:
**suggestion:** we may remove coverage generation on CI, since we don't upload it anywhere.

Reply:
We are going to address this in a separate PR 👍